### ISSUE TITLE
Bir compiler: error while compiling netcdf (COMPILE-AST CLASP-CLEAVIR-AST:FOREIGN-CALL-POINTER-AST)

### ISSUE THREAD

**Issue by kpoeck** [clasp-developers/clasp#1072]
after loading quicklisp:
```lisp
(ql:quickload :netcdf :verbose t)
....
;   (DEFCFUN ("nc_inq_libvers" INQ-LIBVERS) ...)
; caught COMMON-LISP:ERROR:
;   No applicable methods in required group PRIMARY for generic function COMPILE-AST
;   Given arguments: (#<CLASP-CLEAVIR-AST:FOREIGN-CALL-POINTER-AST>
;                     #<CLEAVIR-AST-TO-BIR:INSERTER>
;                     #<CLASP-CLEAVIR::CLASP-64BIT>)
;     at /Users/karstenpoeck/quicklisp/local-projects/cl-netcdf/src/netcdf.lisp 896:0
; 

Condition of type: SIMPLE-ERROR
No applicable methods in required group PRIMARY for generic function COMPILE-AST
Given arguments: (#<CLASP-CLEAVIR-AST:FOREIGN-CALL-POINTER-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
Available restarts:
(use :r1 to invoke restart 1, etc.)

1. (RETRY) Retry compiling #<CL-SOURCE-FILE "netcdf" "netcdf">.
2. (ACCEPT) Continue, treating compiling #<CL-SOURCE-FILE "netcdf" "netcdf"> as having been successful.
3. (RETRY) Retry ASDF operation.
4. (CLEAR-CONFIGURATION-AND-RETRY) Retry ASDF operation after resetting the configuration.
5. (RETRY) Retry ASDF operation.
6. (CLEAR-CONFIGURATION-AND-RETRY) Retry ASDF operation after resetting the configuration.
7. (ABORT) Give up on "netcdf"
8. (RESTART-TOPLEVEL) Go back to Top-Level REPL.

  4: l_NO-REQUIRED-METHOD^CLOS^FN^^
   5: (CLOS::NO-REQUIRED-METHOD #<STANDARD-GENERIC-FUNCTION CLEAVIR-AST-TO-BIR:COMPILE-AST> CLOS::PRIMARY #<CLASP-CLEAVIR-AST:FOREIGN-CALL-POINTER-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
   8: ((LAMBDA (&REST CLOS::VASLIST-ARGS)) #<CLASP-CLEAVIR-AST:FOREIGN-CALL-POINTER-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
  11: l_PERFORM-OUTCOME^CLOS^FN^^
  14: l_DISPATCH-MISS^CLOS^FN^^
  23: l_COMPILE-ARGUMENTS^CLEAVIR-AST-TO-BIR^FN^^
  24: (CLEAVIR-AST-TO-BIR:COMPILE-ARGUMENTS (#<CLASP-CLEAVIR-AST:FOREIGN-CALL-POINTER-AST> #<CLASP-CLEAVIR-AST:PRECALC-CONSTANT-REFERENCE-AST>) #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT> (:OBJECT))
  25: l_COMPILE-AST^CLEAVIR-AST-TO-BIR^((CALL-AST T T))^METHOD^^
  26: ((METHOD CLEAVIR-AST-TO-BIR:COMPILE-AST (CLEAVIR-AST:CALL-AST T T)) #<CLEAVIR-AST:CALL-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
  27: l_COMPILE-AST^CLEAVIR-AST-TO-BIR^((:AROUND) (AST T T))^METHOD^^
  28: ((METHOD CLEAVIR-AST-TO-BIR:COMPILE-AST (:AROUND) (CLEAVIR-AST:AST T T)) #<FUNCTION (METHOD CLEAVIR-AST-TO-BIR:COMPILE-AST (CLEAVIR-AST:CALL-AST T T))> #<CLEAVIR-AST:CALL-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
  31: (CLOS::EMF-FROM-CONTF.LAMBDA #<CLEAVIR-AST:CALL-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
  36: l_COMPILE-AST^CLEAVIR-AST-TO-BIR^((PROGN-AST T T))^METHOD^^
  37: ((METHOD CLEAVIR-AST-TO-BIR:COMPILE-AST (CLEAVIR-AST:PROGN-AST T T)) #<CLEAVIR-AST:PROGN-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
  38: l_COMPILE-AST^CLEAVIR-AST-TO-BIR^((:AROUND) (AST T T))^METHOD^^
  39: ((METHOD CLEAVIR-AST-TO-BIR:COMPILE-AST (:AROUND) (CLEAVIR-AST:AST T T)) #<FUNCTION (METHOD CLEAVIR-AST-TO-BIR:COMPILE-AST (CLEAVIR-AST:PROGN-AST T T))> #<CLEAVIR-AST:PROGN-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
  42: (CLOS::EMF-FROM-CONTF.LAMBDA #<CLEAVIR-AST:PROGN-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
  47: l_COMPILE-AST^CLEAVIR-AST-TO-BIR^((BLOCK-AST T T))^METHOD^^
  48: ((METHOD CLEAVIR-AST-TO-BIR:COMPILE-AST (CLEAVIR-AST:BLOCK-AST T T)) #<CLEAVIR-AST:BLOCK-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
  49: l_COMPILE-AST^CLEAVIR-AST-TO-BIR^((:AROUND) (AST T T))^METHOD^^
  50: ((METHOD CLEAVIR-AST-TO-BIR:COMPILE-AST (:AROUND) (CLEAVIR-AST:AST T T)) #<FUNCTION (METHOD CLEAVIR-AST-TO-BIR:COMPILE-AST (CLEAVIR-AST:BLOCK-AST T T))> #<CLEAVIR-AST:BLOCK-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
  53: (CLOS::EMF-FROM-CONTF.LAMBDA #<CLEAVIR-AST:BLOCK-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
.....
````

**Comment by kpoeck** [comment]
same for `(ql:quickload :trivial-features-tests :verbose nil)`

**Comment by kpoeck** [comment]
and for `(ql:quickload :cffi-tests :verbose nil)`

**Comment by Bike** [comment]
should be fixed by 7eca5f804f6cfea7c68235132d43db49d5fc4c76

**Comment by kpoeck** [comment]
fixed as announced
